web-components/src/components/tabs/scss/tabs.scss

```diff
@@ -62,6 +63,10 @@
       }
     }
   }
+
+  md-menu-overlay::part(overlay) {
+    top: -13px;
```

Comment:
nit: maybe set this value in a variable to explain, like `$tab-menu-manual-offset`

Reply:
Yes, Thanks! Good idea!